pkg/cmd/pipeline/describe.go

```diff
@@ -65,9 +65,12 @@ func describeCommand(p cli.Params) *cobra.Command {
 	f := cliopts.NewPrintFlags("describe")
 
 	c := &cobra.Command{
-		Use:          "describe",
-		Aliases:      []string{"desc"},
-		Short:        "Describes a pipeline in a namespace",
+		Use:     "describe",
```

Comment:
missing `go fmt` ?

Reply:
:thinking: may be or may not be

I tried 

```
✘-INT ~/work/cli [fix-help L|⚑ 10] 
19:43 $ go fmt  pkg/cmd/pipeline/describe.go
✔ ~/work/cli [fix-help L|⚑ 10] 
19:43 $ go fmt $(go list ./... | grep -v /vendor/)
✔ ~/work/cli [fix-help L|⚑ 10] 
19:43 $ git status
On branch fix-help
nothing to commit, working tree clean
✔ ~/work/cli [fix-help L|⚑ 10] 
```

Im missing something?

Reply:
`go fmt -s` I think ?

Reply:
its not formatting. I guess previous diff (in red) got dropped and new format is right.